day7/GroceryStoreInventory.java

```diff
@@ -0,0 +1,66 @@
+package day7;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class GroceryStoreInventory {
+    private static void displayInventory(HashMap<String, Integer> stock) {
+        for(Map.Entry<String, Integer> entry : stock.entrySet()){
+            System.out.println("Name: " + entry.getKey()+","+"Stock Available is "+entry.getValue());
+        }
+    }
+    public static void main(String[] args) {
+        HashMap<String, Integer> stock = new HashMap<>();
```

Comment:
Use Generic for HashMap, Replace `HashMap<String, Integer>` with `Map<String, Integer>`. it is generally better to use interface `Map `instead of concrete implementation `"HashMap"` for flexibility.